test/browser/custom-elements.js

```diff
@@ -223,7 +223,7 @@ describe('Custom Elements', function () {
       };
     });
     // Make VNode with a string as child
-    let vtree$ = Rx.Observable.just(h('myelement'));
+    let vtree$ = Rx.Observable.just(h('div', h('myelement')));
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).